lua/telescope/pickers/entry_display.lua

```diff
@@ -3,6 +3,76 @@ local strings = require "plenary.strings"
 local entry_display = {}
 entry_display.truncate = strings.truncate
 
+---@tag telescope.pickers.entry_display
+
+---@brief [[
+--- Entry Display takes care of formatting the text displayed for each entry.
+--- This includes adding highlight colors.
+---
+--- Most pickers will call the create function only once.
+--- The configuration.items passed in will provide the column
+--- widths which will normally be the same for every entry.
+--- This is why a single call is made to create.
```

Comment:
See above but basically we generate a custom function and we only need to do it once. You can do it n times of course but thats just waist of performance :laughing: